recipes/lely-core/all/conanfile.py

```diff
@@ -0,0 +1,242 @@
+import os
+
+from conan import ConanFile
+from conan.tools.gnu import AutotoolsToolchain, Autotools
+from conan.tools.layout import basic_layout
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.tools.files import (
+    export_conandata_patches,
+    apply_conandata_patches,
+    get,
+    copy,
+    rmdir,
+    rm,
+)
+from conan.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.53.0"
+
+
+class LelyConan(ConanFile):
+    name = "lely-core"
+
+    # Optional metadata
+    license = "Apache"
```

Comment:
```suggestion
    license = "Apache-2.0"
```
Please, always use license SPDX short-name only. https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/conanfile_attributes.md#license-attribute